src/main/java/mate/academy/dao/CinemaHallDao.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.dao;
+
+import java.util.List;
+import java.util.Optional;
+import mate.academy.model.CinemaHall;
+
+public interface CinemaHallDao {
+    CinemaHall add(CinemaHall cinemaHall);
+
+    Optional<CinemaHall> get(Long id);
+
+    List<CinemaHall> getAll();
+
+    Boolean update(CinemaHall cinemaHall);
+
+    Boolean delete(Long id);
```

Comment:
Consider changing the return type of the `delete` method to `void` if the operation's success can be determined by exceptions or other means. Returning a `Boolean` might not be necessary.